src/Gaufrette/Adapter/Local.php

```diff
@@ -91,7 +95,7 @@ public function rename($sourceKey, $targetKey)
      */
     public function exists($key)
     {
-        return is_file($this->computePath($key));
+        return file_exists($this->computePath($key));
```

Comment:
We don't aim to support directory as there's pretty much only Local adapter that has native support for it, thus a "directory" can't exist. What we'll rather support is a mock of it: directory will be only prefixes, so your fix is really valid as we need to delete directory recursively.

In the case of this specific change I'd prefer to not introduce it as it makes an unnecessary extra call for something we don't care.